packaging/windows/registrykeys.wxs

```diff
@@ -25,7 +25,6 @@
         </RegistryKey>
 
         <Environment Id="E_PATH" Name="PATH" Value="[DOTNETHOME]bin" Part="last" Action="set" System="yes" />
-        <Environment Id="E_DOTNET_HOME" Name="DOTNET_HOME" Value="[DOTNETHOME]" Part="all" Action="set" System="yes" />
```

Comment:
That and the Bundle. Technically, the MSI one is hidden but I would align it with what is in the bundle to make it all clear how the install folder flows in.